src/Microsoft.AspNetCore.WebUtilities/WebEncoders.cs

```diff
@@ -305,7 +305,7 @@ public static int Base64UrlEncode(byte[] input, int offset, char[] output, int o
 
         /// <summary>
         /// Get the minimum output <c>char[]</c> size required for encoding <paramref name="count"/>
-        /// <see cref="byte"/>s with the <see cref="Base64UrlEncode(byte[], int, int, char[], int)"/> method.
+        /// <see cref="byte"/>s with the <see cref="M:Base64UrlEncode"/> method.
```

Comment:
`Base64UrlEncode(byte[], int, char[], int, int)`. Again, the problem was in the signature and not the reference to one overload.